### PR TITLE
chore(cli) Port apps, deprecate, and history

### PR DESCRIPTION
**Pull request by xavdid**
A couple of related minor fixes, but otherwise faithful recreations of the old commands. Also cleaned up the help text a bit.

### CODE REVIEW

File: packages/cli/src/oclif/commands/apps.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    this.log('Here are all the apps you have write access to:\n');
+
+    this.logTable({
+      rows: apps,
+      headers: [
+        ['Title', 'title'],
+        ['Unique Slug', 'key'],
+        ['Date Created', 'date'],
+        ['Linked?', 'linked']
```
[the remark this answers is not in the document]
Comment (xavdid) [reply packages/cli/src/oclif/commands/apps.js]: Ah, it was intended (to better communicate that that field was a bool) but it does affect how the json comes out. Not important, I'll revert.

File: packages/cli/src/oclif/commands/deprecate.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+const BaseCommand = require('../ZapierBaseCommand');
+const { buildFlags } = require('../buildFlags');
+
+const { callAPI, getLinkedApp } = require('../../utils/api');
+
+class DeprecateCommand extends BaseCommand {
+  async perform() {
```
[the remark this answers is not in the document]
Comment (xavdid) [reply packages/cli/src/oclif/commands/deprecate.js]: yeah fair point. I was noticing a lot of commands call `checkCredentials` when they don't necessarily need to - it's an extra API call where a subsequent one would correctly return a 401 if auth is messed up. Thought we might be able to get a nice little speed boost by reducing the number of http calls. But, given the current error handling setup, it does make the message a little more confusing. I'll revert for now and we can take a closer look when we get closer to v9.

Same thing goes for `getLinkedApp`, which is typically used to fetch the app's title. that's pretty cacheable though, so another potential speed boost there.

**Comment by xavdid** [comment]
Cool, cleaned those all up in 187b27d. 

Re: minor style consistency. I'm totally into it and It's fine to fix now, but I think our time will be better spent making some choices and doing one big pass once all the code is re-written rather than fixing it along the way.

**Comment by eliangcs** [comment]
> our time will be better spent making some choices and doing one big pass once all the code is re-written

@xavdid sure thing! Thanks for letting me know. Now we're on the same page.

**Comment by xavdid** [comment]
For sure! Your callouts were spot on, I think it's just going to be easier at the end. 

I'm excited to get there!

**Pull request merged**